### PR TITLE
PCX-3180 fetches the versions

### PR DESCRIPTION
**Pull request by dixy-xavier**
## Jira ticket
[PCX-3180](https://optile.atlassian.net/browse/PCX-3180)  

## What
- Adds API request functions
- Fetches the versions of `sandbox` and `live`
- Code format

## Type of change
- New feature (non-breaking change which adds functionality)
- Code format

### BEFORE
image

### AFTER
image

### CODE REVIEW

File: src/services/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import {apiRequest} from './request';
+
+const getLiveVersion = async () => {
+  try {
+    const response = await apiRequest({
+      config: {
+        headers: {'Content-Type': 'text/plain'},
+        method: 'GET',
+      },
+      url: `https://resources.live.oscato.com/paymentpage/v3/version`,
```
Comment (ShahrozSa-payo) [inline src/services/index.js:10]: Is that a new api or an existing one?

File: src/services/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      return response.text();
+    } else {
+      console.error(response);
+    }
+  } catch (error) {
+    console.error(error);
+  }
+};
+
+const getTestVersion = async () => {
```
Comment (AnwerAR) [inline src/services/index.js:22]: Only URL is different between `getLiveVersion` and `getTestVersion`. It would be nice to merge these two methods.

**Review by ShahrozSa-payo — APPROVED** [review]

File: src/services/index.js (hunk 1 above)
Comment (dixy-xavier) [reply src/services/index.js]: @ShahrozSa-payo I don’t see how it's relevant here?

File: src/services/index.js (hunk 2 above)
Comment (dixy-xavier) [reply src/services/index.js]: @AnwerAR done

**Review by AnwerAR — APPROVED** [review]

File: src/services/index.js (hunk 1 above)
Comment (ShahrozSa-payo) [reply src/services/index.js]: I'm not aware of this api, just asking is that an existing one or a new one created just for this ticket

Comment (dixy-xavier) [reply src/services/index.js]: No. I think this API was there before

**Pull request merged**